app/src/main/java/com/rosario/covid19/ui/model/MapperReport.kt

```diff
@@ -0,0 +1,24 @@
+package com.rosario.covid19.ui.model
+
+import com.rosario.covid19.data.model.DataResponse
+import com.rosario.covid19.util.Mapper
+
+class MapperReport : Mapper<DataResponse, ReportViewData>() {
+    override fun mapper(inputElement: DataResponse): ReportViewData {
+        val reportViewData = ReportViewData()
```

Comment:
Why did you create a new object and then fill it with the data instead of return the new instance directly?